python/tests/tensor.py

```diff
@@ -285,10 +277,10 @@ def test_keys_to_samples(self, tensor):
         )
         assert_equal(gradient.data, expected)
 
-    def test_keys_to_samples_unsorted(self, tensor):
-        tensor = tensor.keys_to_samples("key_2", sort_samples=False)
+    def test_keys_to_samples_unsorted(self, tensor_map):
```

Comment:
should we name the fixture `tensor` instead? Makes for simpler code everywhere

Reply:
yes, we can.